pycytominer/cyto_utils/collate.py

```diff
@@ -99,7 +99,7 @@ def collate(
 
             remote_aggregated_file = f"{aws_remote}/backend/{batch}/{plate}/{plate}.csv"
 
-            sync_cmd = f'aws s3 sync --exclude "*" --include "*/Cells.csv" --include "*/Nuclei.csv" --include "*/Cytoplasm.csv" --include "*/Image.csv" {remote_input_dir} {input_dir}'
+            sync_cmd = f'aws s3 sync --exclude * --include */Cells.csv --include */Nuclei.csv --include */Cytoplasm.csv --include */Image.csv {remote_input_dir} {input_dir}'
```

Comment:
```suggestion
            sync_cmd = f"aws s3 sync --exclude * --include */Cells.csv --include */Nuclei.csv --include */Cytoplasm.csv --include */Image.csv {remote_input_dir} {input_dir}"
```

For [`black`](https://github.com/psf/black) formatting. Do you know if this file has already been blacked?

Eventually (been on the TODO list for a while!) we'll have autoformatting enabled in this repo, but until then we've been formatting manual file-by-file. Can you run `black` on this file?

Good to merge after 👍